test/callbacks.jl

```diff
@@ -37,8 +37,8 @@ using Dash
     response = HTTP.handle(handler, request)
     @test response.status == 200
     resp_obj = JSON2.read(String(response.body))
-    @test !in(:multi, keys(resp_obj))
-    @test resp_obj.response.props.children == "test"
+    @test in(:multi, keys(resp_obj))    
```

Comment:
FYI I see a few trailing whitepaces added in this PR (and a few others removed) - it's a little easier to review if we never have these, do you use an editor that can be set to automatically strip these on saving?